news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
+date: 2017-07-xx xx:xx:xx +0000
+tumblr_url: no idea what to do with these fields
+---
+
+Have you ever thought: “Wow, I’m bad at mapping”? Does the quote “So let’s be honest, you suck at mapping” (pishifat, 2016)  strike a personal chord with you? Then the Community Mentorship Program is the place for you!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/ZXakpTkXEC0" frameborder="0" allowfullscreen></iframe>
+
+[seems to need a banner of sorts?]
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles each lasting approximately 10 weeks. Aspiring mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
+
+
+What is the Community Mentorship Program?
```

Comment:
this is well written and i have no qualms with it, but it's a lot of words for a newsposts article that is essentially introducing the mentor program and advertising that applications are open for it. maybe have this in the thread you link to, and shorten the post down as a whole to basically:

* this is what the program does and what activities it offers
* this is who it's intended for
* this is how to apply
* this is when dates end for applying
* go fuckin apply

also nobody cares about the catch integrations as that involves exactly nobody not already in the program. newsposts are general/all reach, so try and not have specific content referenced that most people will not care/do not know about